play.py

```diff
@@ -0,0 +1,89 @@
+class Game:
+    def __init__(self, board, ui):
+        self.ui = ui
+        self.board = board
+
+    def play(self):
+        self.ui.display("Welcome to Battleship")
+        self.ui.display("Take your best shot")
+        self.ui.display(self.board.format())
+        valid_spot = ValueError
+        while valid_spot == ValueError:
+            valid_spot =  self.board.validate(self.ui.get_input('>>'))
+        self.board.update(valid_spot)
+        self.ui.display(self.board.format())
+
+class TerminalUi:
+    def display(self, message):
+       print(message)
+
+    def get_input(self, prompt_string):
+        response = input(prompt_string)
+        return response
+
+class Board:
```

Comment:
I wonder if the responsibilities for maintaining state and creating an output string representing that state belong in the same class.